initialize - failed to get version: terraform

### Server Version
<!--
Run `terraform-ls --version` to show the version, and paste the result between the ``` marks below.
If you are not running the latest version, please try upgrading because your issue may have already been fixed.
-->
```
terraform-ls --version
0.3.1
```

### Terraform Version
<!--
Run `terraform -v to show the version, and paste the result between the ``` marks below.
Make sure you are running the same binary that the language server would normally pick up from $PATH
if you have more than one version installed on your system
-->
```
terraform -v
Terraform v0.12.26
```

### Client Version
<!--
Please share what IDE and/or plugin which interacts with the server
e.g. Sublime Text (LSP plugin) v0.9.7
-->
```
VSCode with the new terraform extension (2.0.0-rc1)
```

### Log Output
<!--
Full debug output can be obtained by launching server with particular flag (e.g. -log-file)
Please follow instructions in docs/TROUBLESHOOTING.md

Please create a GitHub Gist containing the debug output. Please do *NOT* paste
the debug output in the issue, since it may be long.

Debug output may contain sensitive information. Please review it before posting publicly, and if you are concerned feel free to encrypt the files using HashiCorp's GPG public key published at
https://www.hashicorp.com/security#secure-communications
-->
```
Launching language server: c:\Users\XXXXXX\.vscode\extensions\hashicorp.terraform-2.0.0-rc.1/lsp/terraform-ls serve
2020/06/05 13:09:27 service.go:63: Preparing new session ...
2020/06/05 13:09:27 langserver.go:79: Starting server (pid 4828) ...
2020/06/05 13:09:27 server.go:469: Received 1 new requests
2020/06/05 13:09:27 server.go:165: Processing 1 requests
2020/06/05 13:09:27 server.go:223: Checking request for "initialize": 

<SNIP>

2020/06/05 13:09:27 exec.go:178: Starting C:\ProgramData\chocolatey\bin\terraform.exe ["terraform" "version"] in "C:\\Users\\XXXXXX\\AppData\\Local\\Temp"...
2020/06/05 13:09:28 exec.go:144: Waiting for command to finish ...
2020/06/05 13:09:28 rpc_logger.go:45: Error for "initialize" (ID 0): [-32098] failed to get version: terraform (pid 7736) exited (code 1): exit status 1
stdout: ""
stderr: "Couldn't setup logging tempfile: open C:\\WINDOWS\\terraform-log283769091: Access is denied."
2020/06/05 13:09:28 server.go:204: Completed 1 requests [910.1964ms elapsed]
[Error - 1:09:28 PM] Server initialization failed.
  Message: failed to get version: terraform (pid 7736) exited (code 1): exit status 1
stdout: ""
stderr: "Couldn't setup logging tempfile: open C:\\WINDOWS\\terraform-log283769091: Access is denied."
  Code: -32098 
2020/06/05 13:09:28 server.go:469: Received 1 new requests
2020/06/05 13:09:28 server.go:165: Processing 1 requests
2020/06/05 13:09:28 server.go:223: Checking request for "shutdown": 
2020/06/05 13:09:28 rpc_logger.go:29: Incoming request for "shutdown" (ID 1): 
2020/06/05 13:09:28 rpc_logger.go:50: Response to "shutdown" (ID 1): null
2020/06/05 13:09:28 server.go:204: Completed 1 requests [0s elapsed]
2020/06/05 13:09:28 server.go:469: Received 1 new requests
2020/06/05 13:09:28 server.go:165: Processing 1 requests
2020/06/05 13:09:28 server.go:223: Checking request for "exit": 
2020/06/05 13:09:28 rpc_logger.go:29: Incoming notification for "exit": 
```

### Expected Behavior
<!-- What should have happened? -->
On the previous version of the language server, 0.3, terraform would be found and the version obtained.

### Actual Behavior
<!-- What actually happened? -->
lsp complains it is failing to get the terraform version. As well it is trying to write logs to C:\WINDOWS which is denied.

### Steps to Reproduce
<!--
Please list the full steps required to reproduce the issue, for example:
1. Open a folder in IDE XYZ
2. Open file example.tf from that folder
3. Trigger autocompletion on line 5, column 1 (1-indexed)
-->
Open VSCode.

--- a/internal/lsp/completion.go
+++ b/internal/lsp/completion.go
@@ -41,7 +41,7 @@ func CompletionItem(candidate lang.CompletionCandidate, pos hcl.Pos, snippetSupp
 			InsertTextFormat: lsp.ITFSnippet,
 			Detail:           candidate.Detail(),
 			Documentation:    doc,
-			TextEdit: textEdit(candidate.Snippet(), pos),
+			TextEdit:         textEdit(candidate.Snippet(), pos),
 		}
 	}
 
@@ -51,7 +51,7 @@ func CompletionItem(candidate lang.CompletionCandidate, pos hcl.Pos, snippetSupp
 		InsertTextFormat: lsp.ITFPlainText,
 		Detail:           candidate.Detail(),
 		Documentation:    doc,
-		TextEdit: textEdit(candidate.PlainText(), pos),
+		TextEdit:         textEdit(candidate.PlainText(), pos),
 	}
 }
 
@@ -60,12 +60,12 @@ func textEdit(te lang.TextEdit, pos hcl.Pos) *lsp.TextEdit {
 	if rng == nil {
 		rng = &hcl.Range{
 			Start: pos,
-			End: pos,
+			End:   pos,
 		}
 	}
 
 	return &lsp.TextEdit{
-			NewText: te.NewText(),
-			Range: hclRangeToLSP(*rng),
-		}
+		NewText: te.NewText(),
+		Range:   hclRangeToLSP(*rng),
+	}
 }

--- a/internal/terraform/exec/exec.go
+++ b/internal/terraform/exec/exec.go
@@ -108,10 +108,8 @@ func (e *Executor) cmd(args ...string) (*command, error) {
 	// so we don't need to ask checkpoint for upgrades.
 	cmd.Env = append(cmd.Env, "CHECKPOINT_DISABLE=1")
 
-	for _, key := range passthroughEnvVars {
-		if value := os.Getenv(key); value != "" {
-			cmd.Env = append(cmd.Env, key+"="+value)
-		}
+	for _, envVar := range passthroughEnvVars {
+		cmd.Env = append(cmd.Env, envVar)
 	}
 
 	if e.execLogPath != "" {

